### PR TITLE
[Samples] Better muting remote audio toggle + correct MediaError for getUserMedia

### PR DESCRIPTION
**Pull request by taymoork2**
# Pull Request Template

## Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes # (issue)

## Type of Change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Test Coverage

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Test A
- [ ] Test B

**Test Configuration**:
* SDK Version
* Node/Browser Version
* NPM Version

## Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: packages/node_modules/samples/browser-single-party-call-with-mute/app.js [hunk 1]
```diff
@@ -303,7 +303,7 @@ document.getElementById('dialer').addEventListener('submit', (event) => {
         return joinMeeting(meeting);
       });
     })
-    .catch((error) => {
+    .catch(({error}) => {
```
Comment (adamweeks) [inline packages/node_modules/samples/browser-single-party-call-with-mute/app.js]: What is causing the error object to have its own error object?

Comment (taymoork2) [reply packages/node_modules/samples/browser-single-party-call-with-mute/app.js]: It seems that it wraps an error in a MediaError, which it self is an extension of the Error object

```js
 new MediaError(new Error({
	name: err.name,
    localMediaFailed: true,
    cameraFailed: err === mediaConfig.permissionDenied
}), 'Local Get User Media failed')
```

https://github.com/webex/webex-js-sdk/blob/master/packages/node_modules/@webex/plugin-meetings/src/media/index.js#L339

https://github.com/webex/webex-js-sdk/blob/master/packages/node_modules/@webex/plugin-meetings/src/common/errors/media.js

Comment (adamweeks) [reply packages/node_modules/samples/browser-single-party-call-with-mute/app.js]: img

Comment (adamweeks) [reply packages/node_modules/samples/browser-single-party-call-with-mute/app.js]: We need to fix the code for the error generation and not the displaying of the error. Users shouldn't need to know to deconstruct multiple levels of errors to get an error message.

**Pull request merged**